<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="StatusLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>601</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Status Label - display who's turn/ lose messsage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RedButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 24, 24);</string>
    </property>
    <property name="text">
     <string>Red Button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BlueButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(11, 48, 255);</string>
    </property>
    <property name="text">
     <string>Blue Button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Button</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>420</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSImon">
    <property name="title">
     <string>SImon</string>
    </property>
   </widget>
   <addaction name="menuSImon"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
